app/models/statistics/total_savings.rb

```diff
@@ -0,0 +1,27 @@
+class Statistics::TotalSavings
+  def to_s
+    if completed_auctions.count > 0
+      Currency.new(total_starting_price - total_winning_bid_amount).to_s
+    else
+      'n/a'
+    end
+  end
+
+  private
+
+  def total_starting_price
```

Comment:
I think it's fair to call it savings as the customer agency is offering the
work at the starting price (we have not started all auctions at $3500 by
convention, and we actually ask customers to consider a fair starting price
at or below $3500), and the final award (the reduced cost as a result of
the auction) is less than that @harrisj. We have discussed this in previous
grooming sessions but would be happy to revisit with the team.

On Tue, Aug 2, 2016 at 8:55 PM, Robb Shecter <notifications@github.com>
wrote:

> In app/models/statistics/total_savings.rb
> <https://github.com/18F/micropurchase/pull/994#discussion_r73277294>:
>
> > @@ -0,0 +1,27 @@
> > +class Statistics::TotalSavings
> > +  def to_s
> > +    if completed_auctions.count > 0
> > +      Currency.new(total_starting_price - total_winning_bid_amount).to_s
> > +    else
> > +      'n/a'
> > +    end
> > +  end
> > +
> > +  private
> > +
> > +  def total_starting_price
>
> Yep, the math looks good to me. Mathematical average which is just one
> "measure of central tendency" and in the future we can change this if
> another gives better business intelligence.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/micropurchase/pull/994/files/c70f3522ed10f4362fc8f3d98a41a1d8a5117d03#r73277294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGHNbSF3X63sdFrcZV5oQSX3cxugI1Mks5qcBEfgaJpZM4JbKkH>
> .
>



-- 
Michael Torres
Product Lead, Micropurchase Platform
18F
(415) 920-3205